### ISSUE TITLE
[Design System] Améliorer les couleurs du thème

### ISSUE THREAD

**Issue by liliced** [betagouv/mon-entreprise#4507]
Les couleurs du thème gagneraient à être légèrement revues afin d'en améliorer le contraste. Par exemple :
- Le contraste `employeur[200]` VS `employeur[700]` < 7 (=> fail du WCAG AAA), idem pour `particulier`, `independant` et `marin`, de même pour les teintes 200 et 600 (= max, voir plus bas) de `success` et `info`. Pour les autres couleurs c'est ok et ce rapport est utilisé dans les tags (qui sont utilisés avec toutes ces couleurs dans d'autres composants).
- Pour la couleur `info`, même le rapport entre la teinte la plus claire et la teinte la plus sombre ne passe pas ce critère.

Par ailleurs, les couleurs "info", "error" et "success" n'ont que 6 nuances alors que les autres en ont 8, ce qui ajoute de la gymnastique dans leur utilisation.
